python_scripts/logistic_regression.py

```diff
@@ -124,7 +124,10 @@ def plot_decision_function(fitted_classifier, range_features, ax=None):
 ax = sns.scatterplot(
     data=penguins_test, x=culmen_columns[0], y=culmen_columns[1],
     hue=target_column, palette=["tab:red", "tab:blue"])
-_ = plot_decision_function(logistic_regression, range_features, ax=ax)
+plot_decision_function(logistic_regression, range_features, ax=ax)
+plt.show()
+accuracy = logistic_regression.score(data_test, target_test)
```

Comment:
We can move the computation of the accuracy score in the cell above just after the `fit` and let the plotting to be as-is in a single cell.